src/njoy21/input/PLOTR/src/readCurves.hpp

```diff
@@ -0,0 +1,15 @@
+template< typename Istream >
+static std::vector< std::pair< Card2, Curves > >
+ readCurves( Istream& is, Argument< PLOTR::Card1::Lori>& lori )
+{
+  std::vector< std::pair< Card2, Curves > > curveList;
+  Card2 tmp2( is, lori );
```

Comment:
I'm not sure why this compiles. `Card2` only takes one argument for it's constructor. What does it do with `lori`?

Reply:
Definitely takes two.  Uses the page orientation to set limits for the size of the plot, etc.

Reply:
I just don’t know where that constructor is defined. I tried looking for it, but couldn’t.

Reply:
Constructor is found [here](https://github.com/njoy/NJOY21/blob/feature/PLOTR/src/njoy21/input/PLOTR/Card2.hpp#L23).

Reply:
Thanks for pointing that out. Sorry I'm so blind.